arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -1384,3 +1393,65 @@ void DatabaseFeature::enableDeadlockDetection() {
     vocbase->_deadlockDetector.enabled(true);
   }
 }
+
+std::pair<std::string /*sharding*/, std::size_t /*replication*/>
+arangodb::getOneShardOptions(std::string const& collectionName, VPackSlice options) {
+  // Invalid options will be silently ignored. Default values will be used
+  // instead.
+  //
+  // This Function will be called twice - the second time we do not run in
+  // the risk of consulting the ClusterFeature, because defaults are provided
+  // during the first call.
+
+  TRI_ASSERT(!collectionName.empty());
+  bool systemCol = TRI_vocbase_t::IsSystemName(collectionName);
+  std::size_t replicationFactor = systemCol ? 2 : 1; // real default will be read
```

Comment:
why do we initialize the replication factor here manually first, and not directly read initialize it from the ClusterFeature?